edxval/models.py

```diff
@@ -518,6 +518,11 @@ def create_or_update(cls, video, language_code, metadata, file_data=None):
                     )
                     raise
 
+            # Delete the existing transcript file from storage(for example, AWS S3)
+            if not created and old_transcript_path is not None:
```

Comment:
we don't need to check for `old_transcript_path is not None` if we go with above comment.